File: benchmark.js
function findPrimes() {
  for (let i = 2; i <= 100000; i++) {
    if (isPrime(i)) {
      console.log(i);
    }
  }
}

function isPrime(num) {
  if (num < 2) return false;
  for (let i = 2; i <= Math.sqrt(num); i++) {
    if (num % i === 0) return false;
  }
  return true;
}

findPrimes();

File: benchmark.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool isPrime(int num) {
    if (num < 2) return false;
    int limit = sqrt(num);
    for (int i = 2; i <= limit; i++) {
        if (num % i == 0) return false;
    }
    return true;
}

void findPrimes() {
    for (int i = 2; i <= 100000; i++) {
        if (isPrime(i)) {
            printf("%d\n", i);
        }
    }
}

int main() {
    findPrimes();
    return 0;
}
